Documentation/cmdref/cilium-operator-aws.md

```diff
@@ -15,52 +15,55 @@ cilium-operator-aws [flags]
-      --identity-gc-interval duration           GC interval for security identities (default 15m0s)
-      --identity-heartbeat-timeout duration     Timeout after which identity expires on lack of heartbeat (default 30m0s)
-      --ipam string                             Backend to use for IPAM (default "eni")
-      --k8s-api-server string                   Kubernetes API server URL
-      --k8s-client-burst int                    Burst value allowed for the K8s client
-      --k8s-client-qps float32                  Queries per second limit for the K8s client
-      --k8s-heartbeat-timeout duration          Configures the timeout for api-server heartbeat, set to 0 to disable (default 30s)
-      --k8s-kubeconfig-path string              Absolute path of the kubernetes kubeconfig file
-      --k8s-namespace string                    Name of the Kubernetes namespace in which Cilium Operator is deployed in
-      --kvstore string                          Key-value store type
-      --kvstore-opt map                         Key-value store options (default map[])
-      --limit-ipam-api-burst int                Upper burst limit when accessing external APIs (default 4)
-      --limit-ipam-api-qps float                Queries per second limit when accessing external IPAM APIs (default 20)
-      --nodes-gc-interval duration              GC interval for nodes store in the kvstore (default 2m0s)
-      --operator-api-serve-addr string          Address to serve API requests (default "localhost:9234")
-      --operator-prometheus-serve-addr string   Address to serve Prometheus metrics (default ":6942")
-      --parallel-alloc-workers int              Maximum number of parallel IPAM workers (default 50)
-      --subnet-ids-filter strings               Subnets IDs (separated by commas)
-      --subnet-tags-filter stringToString       Subnets tags in the form of k1=v1,k2=v2 (multiple k/v pairs can also be passed by repeating the CLI flag (default [])
-      --synchronize-k8s-nodes                   Synchronize Kubernetes nodes to kvstore and perform CNP GC (default true)
-      --synchronize-k8s-services                Synchronize Kubernetes services to kvstore (default true)
-      --unmanaged-pod-watcher-interval int      Interval to check for unmanaged kube-dns pods (0 to disable) (default 15)
-      --update-ec2-apdater-limit-via-api        Use the EC2 API to update the instance type to adapter limits
-      --version                                 Print version information
+      --aws-instance-limit-mapping map            Add or overwrite mappings of AWS instance limit in the form of {"AWS instance type": "Maximum Network Interfaces","IPv4 Addresses per Interface","IPv6 Addresses per Interface"}. cli example: --aws-instance-limit-mapping=a1.medium=2,4,4 --aws-instance-limit-mapping=a2.somecustomflavor=4,5,6 configmap example: {"a1.medium": "2,4,4", "a2.somecustomflavor": "4,5,6"} (default map[])
```

Comment:
This commit could be squashed with a previous one.

Reply:
Will do that.